backend/src/middleware/notifications/notificationsMiddleware.observing-posts.spec.ts

```diff
@@ -107,20 +107,46 @@ afterAll(async () => {
 
 describe('notifications for users that observe a post', () => {
   beforeAll(async () => {
-    postAuthor = await neode.create('User', {
-      id: 'post-author',
-      name: 'Post Author',
-      slug: 'post-author',
-    })
-    firstCommenter = await neode.create('User', {
-      id: 'first-commenter',
-      name: 'First Commenter',
-      slug: 'first-commenter',
-    })
-    secondCommenter = await neode.create('User', {
-      id: 'second-commenter',
-      name: 'Second Commenter',
-      slug: 'second-commenter',
+    postAuthor = await Factory.build(
+      'user',
+      {
+        id: 'post-author',
+        name: 'Post Author',
+        slug: 'post-author',
+      },
+      {
+        email: 'post-author@example.org',
```

Comment:
Ah, it's back.

Reply:
Yes Neode does not handle to create the second part of this statement - the factory uses it tho for creating an email node for the user. The test was touched in another PR and now requires to use the factory.